holoplot/__init__.py

```diff
@@ -459,6 +467,40 @@ def points(self, x=None, y=None, **kwds):
         """
         return self(x, y, kind='points', **kwds)
 
+    def polys(self, x=None, y=None, c=None, **kwds):
```

Comment:
polys, not polygons?  In most cases the method name echoes the HoloViews object name, but not here?

Reply:
Suppose I should just make it ``polygons``.

Reply:
That would be my vote, but it's your call.